extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/graal/Substitutions.java

```diff
@@ -48,11 +48,11 @@ static void configureLoggerProvider(
         }
     }
 
-    @TargetClass(className = "io.opentelemetry.api.events.GlobalEventEmitterProvider")
-    static final class Target_GlobalEventEmitterProvider {
-        @Substitute
-        public static void set(EventEmitterProvider eventEmitterProvider) {
-            // do nothing. We don't support events yet. Default is EventEmitterProvider.noop()
-        }
-    }
+    //    @TargetClass(className = "io.opentelemetry.api.events.GlobalEventEmitterProvider")
+    //    static final class Target_GlobalEventEmitterProvider {
+    //        @Substitute
+    //        public static void set(EventEmitterProvider eventEmitterProvider) {
+    //            // do nothing. We don't support events yet. Default is EventEmitterProvider.noop()
+    //        }
+    //    }
```

Comment:
I think we should just delete the code instead of commenting it out.

Reply:
yeah... I just noticed the same. Removed.